pkgs/by-name/mc/mcdreforged/package.nix

```diff
@@ -0,0 +1,49 @@
+{
+  lib,
+  fetchFromGitHub,
+  python3,
+  testers,
+  mcdreforged,
+}:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "mcdreforged";
+  version = "2.12.3";
+
+  src = fetchFromGitHub {
+    owner = "Fallen-Breath";
+    repo = "MCDReforged";
+    rev = "refs/tags/v${version}";
+    sha256 = "sha256-MLub++mkkB/jshpHJXtqgIhs7Gcb4jHUyHqGE65S8A8=";
+  };
+
+  disabled = python3.pkgs.pythonOlder "3.8";
```

Comment:
This is irrelevant. We don't even support 3.8.

Reply:
It seems to be a common practice in other python packages. Any related documentation?

Reply:
In python3Packages those are partly from older releases and they are added there sometimes but it is not a hard requirement for EOL python versions. If it is regarding a current python version then it should be added, to avoid the failing builds on hydra.

But since this is a standalone package, the disabled lines are not doing anything, since someone would need to overwrite python3 with python37 or so and that will fail at many other points already. So you can just drop this line with the next PR to the package to keep things clean.